modules/api/src/main/java/org/apache/ignite/app/Ignition.java

```diff
@@ -40,8 +40,20 @@
     /**
      * Starts an Ignite node with an optional bootstrap configuration from an input stream with HOCON configs.
      *
-     * @param name Name of the node. Must not be {@code null}.
-     * @param config Input stream from the node configuration in HOCON format. Can be {@code null}.
+     * @param name    Name of the node. Must not be {@code null}.
+     * @param config  Input stream from the node configuration in HOCON format. Can be {@code null}. Optional node
```

Comment:
```
     * Optional node configuration based on {@link org.apache.ignite.configuration.schemas.runner.NodeConfigurationSchema}
     *                and {@link org.apache.ignite.configuration.schemas.network.NetworkConfigurationSchema}. Following
     *                rules are used for applying the configuration properties:
     *                <ol>
     *                    <li>Specified property overrides the existing one or just apply itself if it wasn't previously specified.</li>
     *                    <li>All non-specified properties either use the previous value or use the default one from the corresponding configuration schema.</li>
     *                </ol>
     *                So that, in case of initial node start (first start ever) specified configuration, supplemented with defaults,
     *                are used. If no configuration was provided, defaults are used for all configuration properties.
     *                In case of node restart, specified properties override existing ones, non specified properties that also
     *                weren't specified previously uses default values. Please pay attention that previously specified properties
     *                are searched in the {@code workDir} specified by the user.
```

Reply:
Fixed with minor changes.